src/guide/ssr/build-config.md

```diff
@@ -0,0 +1,83 @@
+# Build Configuration
```

Comment:
**note**: I still don't like how this file looks like, would be happy to have any improvements suggested